qmlnotifyservice.cpp

```diff
@@ -22,10 +23,12 @@ bool init(QQmlApplicationEngine *engine) {
         return false;
 
     initSNotufyResources();
+    QPointer<HistoryNotificationsModel> historyModel = new HistoryNotificationsModel();
 
     engine->addImportPath(":/");
 
     root->setContextProperty("notificationService", NotificationService::getService());
+    root->setContextProperty("historyNotificationsModel", historyModel);
```

Comment:
All registered object available on the all qml app, this may be issue because SimpleQmlNotify is libarray, and will be included on different apps. 
use the property of the main service model that give access to model